### PR TITLE
0.10.1 Backport and release

### PR DESCRIPTION
**Pull request by Mixih**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Mixih** [comment]
reack c671fc7

**Comment by hemangsk** [comment]
ack 0dc6e2b

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,3 +1,12 @@
+coala-bears 0.10.1
+==================
+
+Bugfixes:
+
+- Remark dependency has been pinned back to ~5.4 because version 6 changes
```
Comment (meetmangukiya) [inline RELEASE_NOTES.rst]: 5 or 5.4 ? :/ ?

**Comment by adtac** [comment]
ack https://github.com/coala/coala-bears/pull/1451/commits/577655212b0b80f2ff79a0c57d28b56acadcdbac

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (adtac) [inline RELEASE_NOTES.rst]: 5 will map to whatever's the latest in 5.x

6 is breaking changes

Comment (Mixih) [inline RELEASE_NOTES.rst]: theoretically 5.4. See original pr for explanation

**Comment by Mixih** [comment]
@rultor merge

**Pull request merged**